### PR TITLE
Redesign event view page

### PR DESCRIPTION
**Pull request by mohamudm1**
Solved issue #567 

Changes made:
Redesigned the view event page complete. You can see what it looked like before and how it looks not below
Even View Page Window
After

### CODE REVIEW

File: app/templates/eventView.html [hunk 1]
```diff
@@ -29,82 +29,94 @@ <h1 id="title">{{page_title}}</h1>
...
-        <p class="heading-size"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</p>
-    {% else %}
-        <p class="heading-size"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</p>
-    {% endif %}
-            <div>
-                <div>
-                    <h3><br>Description:</h3>{{eventData.description}}<br><br>
-                    <div>
+            <div class="row">
+                    <div style="padding-right:150px;"class="col-md-6">
```
Comment (zachneill) [inline app/templates/eventView.html]: The padding-right here gets a bit noticeable on smaller screens and especially mobile. Taking out the padding looked okay. If you still want padding, I'd suggest using rem or similar, not pixels.

File: app/static/css/base.css [hunk 2]
```diff
@@ -31,6 +31,11 @@
 
 .footerlink {
   font-weight: bold;
+
+}
+
+.size{
```
Comment (zachneill) [inline app/static/css/base.css]: The class name "size" is pretty unindicative. A slightly more obvious name would be helpful

File: app/templates/eventView.html [hunk 3]
```diff
@@ -29,82 +29,94 @@ <h1 id="title">{{page_title}}</h1>
...
-                                    {{programManager.user.firstName}} {{programManager.user.lastName}}<br>
-                                    {{programManager.user.email}}
-                                </div>
-                        {% endif %}
-                    </div>
-                </div>
-                <div>
-                    <div class="row">
+
+                <div class=" col-md-6">
```
Comment (zachneill) [inline app/templates/eventView.html]: If you made it this far, might as well remove the added spaces in front of your col-md-6 's. I'm assuming you removed a class in front of it...

**Review by zachneill — DISMISSED** [review]

**Comment by BrianRamsay** [comment]
Let's try putting that first descriptive sentence above the Description and Facilitators headers. I also think the font-size class can go. The smaller text is fine, since that's the default text size we use in the rest of the application

File: app/templates/eventView.html [hunk 4]
```diff
@@ -27,84 +27,89 @@ <h1 id="title">{{page_title}}</h1>
...
-                                            <div><b>{{training}}</b>: Attended</div>
-                                        {% elif participated == 0 %}
-                                            <div><b>{{training}}</b>: Not attended</div>
-                                        {% else %}
-                                            <div><b>{{training}}</b>: Upcoming</div>
-                                        {% endif%}
-                                    {% endif %}
-                                {% endfor %}
+                          {% if programManager %}
+                              <div><h3><br>Program Manager</h3></div>
```
Comment (BrianRamsay) [inline app/templates/eventView.html]: No reason for a div around the header that I can see.

File: app/templates/eventView.html [hunk 5]
```diff
@@ -27,84 +27,89 @@ <h1 id="title">{{page_title}}</h1>
...
-                <div>
-                    <h3><br>Description:</h3>{{eventData.description}}<br><br>
+            <div class="row">
+                    <div class="col-md-6">
+                    <h3><br>Description</h3>
+                    {{eventData.description}}<br><br>
                     <div>
                         {% if eventData.isService %}
                             This event earns service hours.
+                            <br />
```
Comment (BrianRamsay) [inline app/templates/eventView.html]: Put all of these in `<p>` tags, instead of using breaks

File: app/templates/eventView.html [hunk 6]
```diff
@@ -27,84 +27,89 @@ <h1 id="title">{{page_title}}</h1>
...
                           {% endif %}
-                        </div>
-                    </div>
-                </div>
+                      </div>
+                  </div>
             </div>
-            <div class="text-center">
+
+            <div style="padding-top:100px;"class="text-center">
```
Comment (BrianRamsay) [inline app/templates/eventView.html]: We want to avoid style tags, and avoid defining things with pixels

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**